diagnostic.md

```diff
@@ -2,37 +2,51 @@
 
 What are three advantages of using Ember?
 
-```sh
-# your answer here
+```md
+A routing system that allows for a single page application allowing for
+  back functionality and boomarking
+Less AJAX
+More use of the cache because of less AJAX, yielding more responsiveness
 ```
 
 What is the templating system used in Ember? How it is related to
 handlebars?
 
-```sh
-# your answer here
+```md
+Each template is written in Handlebars. The template is the HTML content that
+corresponds to the route.
```

Comment:
Yes. Ember adds to handlebars functionality with helpers like `link-to` and the ability to call component templates by file name.
1/1